starter_code/src/movies.js

```diff
@@ -2,14 +2,102 @@
 
 // Iteration 1: Ordering by year - Order by year, ascending (in growing order)
 
+function orderByYear(arr) {
+  const orderedByYear = arr.slice().sort(function(a, b) {
+    if (a.year === b.year) {
+      return a.title.localeCompare(b.title);
+    }
+    return a.year - b.year;
+  });
+  return orderedByYear;
+}
+
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct
+function howManyMovies(arr) {
+  const howManyStevens = arr.filter(function(movie) {
+    if (
+      movie.director === "Steven Spielberg" &&
+      movie.genre.indexOf("Drama") > -1
```

Comment:
another way of doing this line here would be with a 
```
movie.genre.includes("Drama")
```

The includes method also returns a boolean